config/environments/development.rb

```diff
@@ -1,10 +1,11 @@
+require "active_support/core_ext/integer/time"
+
 # rubocop:disable Metrics/BlockLength
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  # In the development environment your application's code is reloaded on
-  # every request. This slows down response time but is perfect for development
-  # since you don't have to restart the web server when you make code changes.
+  # In the development environment your application's code is reloaded any time
+  # it changes. This slows down response time but is perfect for development
```

Comment:
```suggestion
  # `cache_classes = false` ensures that your application's code is reloaded anytime
  # it changes in the dev environment. This slows down response time, but is perfect for development.
```
Just to make it a little clearer that this comment relates specifically to the `config.cache_classes` setting below!

Reply:
FYI: these are autogenerated by Rails :D I can totally merge it but we tend to leave the default ones, see https://github.com/rails/rails/blob/638b6bd06a25f8f793b49fc33c21479840d16599/railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt#L6-L9

Let me know what you'd prefer

Reply:
OH 🤦🏽‍♀️ hah, in that case you can ignore this. Let's just go with the auto-generated ones! 👍🏽